### ISSUE TITLE
Always random point

### ISSUE THREAD

**Issue by wkronemeijer** [CrumpyOldLord/pws-ts#1]
Current situation:
- Points are generated randomly
- You cannot add your own point set

Change this to:
- Points are randomly generated only once
- Add your own set of points (and save them)
- Generate algorithm statistics with one button (the average of X number of tries with the same set of points, discarding the first one)

**Comment by wkronemeijer** [comment]
> - Generate algorithm statistics with one button (the average of X number of tries with the same set of points, discarding the first one)

Postpone this one for now; multiple graphs is way out as well.
